### DISCUSSION TITLE
target 90

### DISCUSSION THREAD

**Discussion by verberg** [cssbattle/forum#209] [Q&A] [answered]
I found (for me) a new solution for target 90, and I don't see why it gets not 100%. Have you any idea?
Thank's! 

`<style>*>*{color:#F3AC3C;border-radius:50%;margin:150 0-250 0;box-shadow:0-200px 0-100px#1A4341,0-200px 0-75px,0-400px#998235,inset 9in 0#998235,0 0 0 99vw`

**Comment by jacobovillegas** [comment, marked as answer]
Change `inset 9in 0#998235,`

for

`background:#998235`

then

change the `*>*` for `*` and divide every single value in the margin by 2

**Reply by verberg** [reply]
Yes, that's it! thanks very much! But only, when I change` *>*` for `*`
do you know why?

**Reply by SelenIT** [reply]
The only explanation that comes to my mind is rounding errors that might lead to differently placed pixels on the edge of the circle for even and odd values of margins 🤷 maybe it's also worth noting that, in the `*>*` version, replacing `50%` of border radius with `49.8%` also makes the algorithm accept it as 💯

**Comment by rafehimself** [comment]
inset shadow often cause 99.9%, but I don't know why
more over I found that in [#29](https://github.com/cssbattle/forum/discussions/192)  that algorithm accept `background:#1A4341` but doesn't accept shadow ` 0 0 0 9in#1A4341` 
I don't know what is the difference as well ...

**Reply by verberg** [reply]
Thanks for your reply. But it doesn't function. But it does width the solution from jacobovillegas (above).
for #29, I have many solutions width 99.9%, recently one only width radial-gradients, but I don't know why.

**Reply by gunnarbittersmann** [reply]
That’s funny, indeed.

100%: `<body bgcolor=1A434 text=F3AC3C><img style=…;box-shadow:4in+0,0+4in,4in+4in>`

99.9%: `<body text=F3AC3C><img style=…;box-shadow:4in+0,0+4in,4in+4in,0+0+0+9in#1A4341>`

But setting `bgcolor` to a non-empty value (you won’t see any of it) matches

100%: `<body bgcolor=x text=F3AC3C><img style=…;box-shadow:4in+0,0+4in,4in+4in,0+0+0+9in#1A4341>`

Also with CSS (needs to be a valid color value):

100%: `<body text=F3AC3C><img style=background:red;…;box-shadow:4in+0,0+4in,4in+4in,0+0+0+9in#1A4341>`

That Chromium rendering engine is full of surprises.
